internal/stack/compose.go

```diff
@@ -170,18 +170,6 @@ func withDependentServices(services []string) []string {
 	return services
 }
 
-func withIsReadyServices(services []string) []string {
-	if len(services) == 0 {
-		return services // load all defined services
-	}
-
-	var allServices []string
-	for _, aService := range services {
-		allServices = append(allServices, aService, fmt.Sprintf("%s_%s", aService, readyServicesSuffix))
```

Comment:
https://github.com/elastic/elastic-package/blob/main/test/packages/parallel/sql_input/_dev/deploy/docker/docker-compose.yml#L10 hmmm not sure about this

Reply:
Those container definitions are part of the services required for testing (system tests). 

Related to the `status` command, it should not affect since that command is intended to show just information about the containers related to the Stack (Kibana, Package Registry, Elasticsearch, etc.). It uses the docker-compose project from the profile:
https://github.com/elastic/elastic-package/blob/62fe1c966f641e57a8f89df760a7c88e7bd6ec0b/internal/stack/compose.go#L189

About the test packages, not sure what it would be the best option.
Currently, the `servicedeployer` run in detached mode, using `-d` docker-compose flag:
https://github.com/elastic/elastic-package/blob/62fe1c966f641e57a8f89df760a7c88e7bd6ec0b/internal/servicedeployer/compose.go#L109
https://github.com/elastic/elastic-package/blob/62fe1c966f641e57a8f89df760a7c88e7bd6ec0b/internal/servicedeployer/custom_agent.go#L132
https://github.com/elastic/elastic-package/blob/62fe1c966f641e57a8f89df760a7c88e7bd6ec0b/internal/servicedeployer/terraform.go#L145


About the test packages, if `servicedeployer` is not updated with the new flags, those new flags those container should be kept. And it is also run a explicit method to wait for the containers being ready/healthy:
https://github.com/elastic/elastic-package/blob/62fe1c966f641e57a8f89df760a7c88e7bd6ec0b/internal/servicedeployer/terraform.go#L152
https://github.com/elastic/elastic-package/blob/62fe1c966f641e57a8f89df760a7c88e7bd6ec0b/internal/servicedeployer/custom_agent.go#L150

@jsoriano Should `servicedeployer` be updated too (Up options) with these new flags? Or keep the current implementation ? As they are running with `-d` , it looks safe.

About the test package , it could be removed... but in the integrations repository they would keep using that container. Probably, it could be kept the `tianon/true` container to be sure that it is tested also with that. WDYT ?